### PR TITLE
Minor fixes to lecture and assignments

### LINKED ISSUE #323: A few assignment a lecture fixes
There are a couple of minor issues in some of my lectures and assignments that I would like to fix. They are low priority but I'm making a note here so I don't forget.

- [x] Assignment 4, question 3: Make clear that it's possible the final model has no significant effects. Students should indicate this for the visualisation questions if this is true.
- [x] Lecture 9: The `k` in `n/k` for deciding between AIC and AICc refers to the _total_ number of parameter (fixed, random, intercept). Needs to be made more clear.
- [x] Lecture 12: Change calculation of simulated male and female means to use `dplyr` instead of base R
- [x] Assignment 4 has two question 3.
- [x] Remove bit about "whatever plot you want" in assignment 4 question 3b
- [x] Assignment 4 Q3a should say banner petal width
- [x] Assignment 4 Q3b (LM): Clarify "when other variables are in the model"
- [x] Reformat text block in assignment 5
- [x] Assignment 4: Clarify that spatial blocks were added _post-hoc_ and are not a result of the experimental design. Therefore, it make sense to assess whether this random effect is needed (in assignment 5 model selection).
- [x] Assignment 5: Ask for justification for each random effect being included/excluded from the models
- [x] Challenge assignment: Specify # iterations for Q1d

I will add more small issues here as they come up, and submit a single PR towards the end of the course that addresses them.

### PR DESCRIPTION
**Pull request by James-S-Santangelo**
Fixes #323 

- Added a few clarifications to  assignment questions. These clarifications were sent to the class as Emails so I'm cementing them into the assignments now. 
- One clarification to lecture 9, which was also sent to the class via Email.

### CODE REVIEW

File: assignment-05.Rmd [hunk 1]
```diff
@@ -78,26 +119,50 @@ total.abun := b + (a*d) + (d*e*c)
...
+    b. Evaluate the model, bootstrapping confidence intervals for path
+    coefficients with seed #778. Which predictor had the largest _direct_ effect
+    on malaria prevalence? How about _total_ effect? Briefly explain these
+    effects in plain english (1 sentence each). (1 mark)
 
 
-3. In this exercise, we will be investigating the relationship between vector community structure and another commonly used metric of disease risk -- entomoligcal innoculation rate (EIR). EIR is a measure of the number of bites by infectious mosquitoes per person per unit time. We will be making use of the same data from Mbogo _et al._ (2003) as before, only this time, we will start with the long form data. (2.5 marks total)
+3. In this exercise, we will be investigating the relationship between vector
+community structure and another commonly used metric of disease risk --
+entomoligcal innoculation rate (EIR). EIR is a measure of the number of bites by
```
Comment (aays) [inline assignment-05.Rmd]: entomological inoculation?

**Comment by aays** [comment]
Looks great - I just have one tiny typo fix. Will also probably submit a separate PR soon for changes to my lectures/assignments.

**Comment by James-S-Santangelo** [comment]
@aays I'm shadow merging this PR since it's been reviewed and has been sitting a while. I only just got to incorporating your edits.
